source/_docs/autostart/systemd.markdown

```diff
@@ -31,6 +31,7 @@ Description=Home Assistant
 After=network.target
 
 [Service]
+WorkingDirectory=/home/homeassistant
```

Comment:
If systemd run as user this shouldn't be needed as it points to the user's (`User=`) home directory as far as I remember.

Reply:
Would it hurt? I guess not...